test/testdata/rbi/module.rb

```diff
@@ -7,3 +7,7 @@
 T.assert_type!(Module.const_source_location('Zlib'), T.nilable([String, Integer]))
 T.assert_type!(Module.const_source_location('Zlib', true), T.nilable([String, Integer]))
 T.assert_type!(Module.const_source_location(:Zlib, false), T.nilable([String, Integer]))
+T.assert_type!(Module.private_class_method([:foo, :bar]), T::Array[Symbol])
+T.assert_type!(Module.private_class_method(["foo", "bar"]), T::Array[String])
+T.assert_type!(Module.private_class_method(:foo), Symbol)
+T.assert_type!(Module.private_class_method("bar"), String)
```

Comment:
According to the sig, `private_class_method` returns `Module`, so these `assert_type!` calls need to be modified to assert for `Module`.